pages/community/precice-workshop-2024.md

```diff
@@ -259,6 +259,56 @@ The cost of lunch, as well as coffee and snacks is included in the registration
 - 15:00-15:30: ☕️ Coffee break
 - 15:30-17:30: Hands-on user support
 
+## Program details
+
+### User introductions
+
+The official welcome to the workshop on Wednesday 25 will be followed by a round of introductions, so that everyone knows who to talk to during the workshop.
+[Submit](https://ipvs.informatik.uni-stuttgart.de/cloud/s/JWJ4toJF2qCScYL) one PDF slide (16:9, with name `surname-firstname.pdf`) with your name and some key details/pictures about your research till September 23, as we need to prepare a single slideshow.
+
+### World Café
+
+A classic for a preCICE Workshop, the preCICE [World Café](https://en.wikipedia.org/wiki/World_caf%C3%A9_(conversation)) is your opportunity to shape the future of preCICE. Expect topics such as Features, Usability, Community, Funding, Teaching, and This Workshop.
```

Comment:
@uekerman do you want to already mention something about preECO here?

Reply:
We could decide on the topics together on Thursday and update this list (different each year anyway). IIRC, this year we wanted at least two (three?) preECO-related topics:

- Guidelines for adapters
- Guidelines for application cases
- Tooling?

We are also not so many people this year, so we don't need as many tables as last year.

Reply:
@MakisH can you please directly change the topics based on yesterdays discussion?

Reply:
Done